docs/javadoc/2.8/com/fasterxml/jackson/databind/JsonMappingException.html

```diff
@@ -353,7 +353,7 @@ <h3>Method Summary</h3>
 <tr class="altColor">
 <td class="colFirst"><code><a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
 <td class="colLast"><code><strong><a href="../../../../com/fasterxml/jackson/databind/JsonMappingException.html#getPathReference()">getPathReference</a></strong>()</code>
-<div class="block">Method for accesing description of path that lead to the
+<div class="block"><a href="https://www.scaler.com/topics/java/methods-in-java/">Method</a> for accesing description of path that lead to the
```

Comment:
just my opinion but I don't think that we need links to describe what a Java method is - we need to assume a certain level of Java knowledge - otherwise the docs will become very bloated